_po/ja/how-to/laravel.po

```diff
@@ -533,6 +779,29 @@ msgid ""
 "                            </div>\n"
 "                        </form>"
 msgstr ""
+"                    <div>\n"
+"                        <div class=\"flex w-full justify-between items-center s"
+"m:mb-2\">\n"
+"                            <h2 class=\"text-3xl font-extrabold tracking-tight "
+"text-gray-900 sm:text-4xl sm:mb-4\"><a href=\"/\">PGroonga Laravel</a> </h2>\n"
+"                        </div>\n"
+"                        <form action=\"{{ route('posts.search') }}\" method=\"GET"
+"\">\n"
+"                            <div class=\"flex mb-4 justify-between items-center"
+"\">\n"
+"                                <div class=\"block w-3/4\">\n"
+"                                    <div class=\"flex flex-col sm:flex-row just"
+"ify-start items-center pl-2\">\n"
+"                                    <input type=\"search\" name=\"keyword\" class="
+"\"form-control w-full sm:w-5/6 \" type=\"text\" value=\"@if (isset($keyword)) {{ $k"
+"eyword }} @endif\" placeholder=\"{{ __('Enter search keyword') }}\">\n"
+"                                    <button class=\"collapse sm:visible inline-"
+"block align-left text-base sm:w-20 rounded-md border border-gray-700 sm:p-2 sm"
+":ml-4\" type=\"submit\">{{ __('Search') }}</button>\n"
+"                                    </div>\n"
+"                                </div>\n"
+"                            </div>\n"
+"                        </form>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.